client/lib/a11y/use-aria-props.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Internal dependencies
+ */
+import { ARIA_STATES, ARIA_PROPERTIES } from './constants';
+
+/**
+ * Types
+ */
+import type {
+	AriaProps,
+	AriaAttributes,
+	AriaState,
+	AriaProperty,
+	RawAriaState,
+	RawAriaProperty,
+} from './types';
+
+export default function useAriaProps( props: AriaProps ): AriaAttributes {
```

Comment:
Hook that transforms, for instance, `{ hidden: true }` to `{ 'aria-hidden': true }`.